### PR TITLE
About

### PR DESCRIPTION
**Pull request by Guitarninja1**
mine
Screen Shot 2021-08-20 at 9 45 50 AM
Screen Shot 2021-08-20 at 9 46 12 AM
Screen Shot 2021-08-20 at 9 46 26 AM
Screen Shot 2021-08-20 at 9 46 39 AM
Screen Shot 2021-08-20 at 9 47 15 AM
template
Screen Shot 2021-08-20 at 9 48 47 AM
Screen Shot 2021-08-20 at 9 48 19 AM
Screen Shot 2021-08-20 at 9 48 09 AM
Screen Shot 2021-08-20 at 9 47 55 AM
Screen Shot 2021-08-20 at 9 47 43 AM

### CODE REVIEW

**Comment by kalm42** [comment]
The spacing on the left/right boxes is off. In the template the boxes are evenly spaced. Your second and third boxes are touching.

File: views/About.module.css [hunk 1]
```diff
@@ -167,3 +174,57 @@
...
+	border: 2px solid rgba(248, 248, 252, 0.1);
+	position: relative;
+	flex-direction: column;
+	align-items: center;
+	width: 262px;
+	height: 254px;
+	margin-left: 80px;
+}
+.iconblockodd {
+	margin: 15px;
```
Comment (kalm42) [inline views/About.module.css:199]: This is why your blocks are not evenly spaced.

**Review by kalm42 — CHANGES_REQUESTED** [review]

File: views/About.module.css (hunk 1 above)
Comment (Guitarninja1) [reply views/About.module.css:199]: mine
Screen Shot 2021-08-20 at 12 58 43 PM

File: views/About.module.css [hunk 2]
```diff
@@ -167,3 +174,60 @@
...
+			360deg,
+			rgba(255, 69, 91, 0.07) 0%,
+			rgba(255, 144, 69, 0) 30%
+		),
+		url(/wierd.svg);
+	background-position: bottom;
+	background-repeat: no-repeat;
+	/* display: flex;
+	flex-direction: row;
+	flex-wrap: wrap; */
```
Comment (kalm42) [inline views/About.module.css:190]: Avoid committing commented out code.

**Review by kalm42 — APPROVED** [review]

File: views/About.module.css (hunk 2 above)
Comment (Guitarninja1) [reply views/About.module.css:190]: i noticed and deleted it. lol

**Pull request merged**